### PR TITLE
Add About Component

### PR DESCRIPTION
**Pull request by Starboy-Sharma**
I have added a description from Readme. Let me know if you want to add anything to the About page.
Issue #27

### CODE REVIEW

File: src/components/About.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { Box, Button, Flex, Heading, Paragraph } from "theme-ui";
+
+export const About = () => {
+  return (
+    <Box>
+      <Heading>About</Heading>
+      <Paragraph variant="block">
```
Comment (braaar) [inline src/components/About.tsx]: I don't think we have any `block` variant for paragraphs defined in the theme 🤔

Comment (Starboy-Sharma) [reply src/components/About.tsx]: I will remove this variant and let you know.

File: src/components/About.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Box, Button, Flex, Heading, Paragraph } from "theme-ui";
+
+export const About = () => {
+  return (
+    <Box>
+      <Heading>About</Heading>
+      <Paragraph>
+        Hi there 👋, Welcome in Jam Assistant. A free open-source web app that
```
Comment (braaar) [inline src/components/About.tsx]:
```suggestion
        Welcome to Jam Assistant, a free open-source web app that
```

Comment (Starboy-Sharma) [reply src/components/About.tsx]: Okay I will update the code and let you know

**Review by braaar — APPROVED** [review]

**Comment by braaar** [comment]
Thanks, @Starboy-Sharma!

**Pull request merged**